### PR TITLE
Fix several issues of the data generator and BanyanDB plugin

### PR DESCRIPTION
**Pull request by hanahmily**

### CODE REVIEW

**Review by kezhenxu94 — APPROVED** [review]

File: oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBConverter.java [hunk 1]
```diff
@@ -272,5 +281,26 @@ public byte[] getBytes(String fieldName) {
...
+            } else if (Long.class.equals(clazz) || long.class.equals(clazz)) {
+                return 0L;
+            } else if (String.class.equals(clazz)) {
+                return "";
+            } else if (Double.class.equals(clazz) || double.class.equals(clazz)) {
+                return 0D;
+            } else if (StorageDataComplexObject.class.isAssignableFrom(clazz)) {
+                return "";
+            } else if (clazz.isEnum()) {
+                return 0L;
```
Comment (wu-sheng) [inline oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBConverter.java:297]: Should not NULL for Enum?

Comment (hanahmily) [reply oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBConverter.java:297]: Yes, the enum is either 0 or 1.

Comment (wu-sheng) [reply oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBConverter.java:297]: Why Enumb is 0 or 1? I am confused.

**Review by wu-sheng — APPROVED** [review]

**Comment by wu-sheng** [comment]
Please check the CI.

**Pull request merged**